package.json

```diff
@@ -18,8 +18,8 @@
     "test": "graph test"
   },
   "dependencies": {
-    "@graphprotocol/graph-cli": "^0.97.0",
-    "@graphprotocol/graph-ts": "0.32.0"
+    "@graphprotocol/graph-cli": "^0.97.1",
+    "@graphprotocol/graph-ts": "^0.32.0"
```

Comment:
next time, please be sure that the dependencies are updated in a different PR to keep them split in case we need to revert